mlflow/langchain/_langchain_autolog.py

```diff
@@ -47,25 +47,28 @@ def _invoke(self, *args, **kwargs):
 
     config = AutoLoggingConfig.init(mlflow.langchain.FLAVOR_NAME)
 
+    logged_model = mlflow.create_logged_model(name="model")
```

Comment:
What happens if this LangChain model was loaded from an existing logged model using `mlflow.langchain.load_model()`? Can we reuse that logged model?

It would be great to make the same thing work for pyfunc inference with a wrapped langchain model loaded via `mlflow.pyfunc.load_model()`

Reply:
Good point. The logged model would create a new run and model.

Reply:
Not sure if this is feasible, but we could attach an attribute to the logged model and checks that attribute when `invoke` is called.